src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2278,6 +2278,7 @@
 helps['tag create'] = """
 type: command
 short-summary: Create tags on a specific resource.
+note: az tag wipes existing tags. To solve this, one can get and buffer existing tags and add new ones to buffer. Then set buffered tags on resource.
```

Comment:
In fact, there is no `note` entry in help. Please add this part to the end of `long-summary`.
For how to write help, please refer to this doc [authoring_help]( https://github.com/Azure/azure-cli/blob/dev/doc/authoring_help.md)